modules/microshift-config-yaml.adoc

```diff
@@ -27,20 +27,32 @@ node:
   hostnameOverride: "" <5>
   nodeIP: "" <6>
 apiServer:
-  subjectAltNames: [] <7>
+  advertiseAddress:
+    - 10.44.0.0/32 <7>
```

Comment:
This object is a string in the config, furthermore this value takes an IP address, the CIDR range will error as it currently stands.

```yaml
apiServer:
  advertiseAddress: "10.44.0.0" <7>
``` 

Note when it comes to the IP address, if the user supplies one , they must make sure that IP address is reachable as described here https://github.com/openshift/openshift-docs/pull/65057/files#diff-73229dc048def016483d7e40c6bc3f7c6015817779b5205bbd5af3592b4e0aa3R56 I'm not sure if we want to make sure that's highlighted more, so users don't get confused if they supply an IP address that's not reachable

Reply:
Maybe we should also add that check to microshift, as part of sanitizing the input.